secrets.sh

```diff
@@ -438,11 +438,11 @@ EOF
 		if [[ $yml =~ ^=.*$ ]]; then
 		    yml="${yml/=/}"
 		fi
-		if [[ $yml =~ ^(.*/)?secrets(\.[^.]+)*\.yaml$ ]]
+		if [[ $yml =~ ^(.*/)?secrets([_.-][^.]+)*\.yaml$ ]]
 		then
 		    decrypt_helper $yml ymldec decrypted
 		    cmdopts+=("$ymldec")
-		    [[ $decrypted -eq 1 ]] && decfiles+=("$ymldec")
+		    [[ $decrypted -eq 1 ]] && decfiles+=("$ymldec") && echo "DECRYPTED $yml temporarily"
```

Comment:
Can you add flag to enable disable this print?

Reply:
Sure. I imagine this is for backward compat, basically by default you don't want the output? I can add a flag `--verbose-decrypt` or `--secrets-verbose` or similar. I don't think it is a good idea to use the verbosity setting of helm command itself, because that would turn on way more output than typically desired. Let me know what you think.